### DISCUSSION TITLE
Welcome to realm-dotnet Discussions!

### DISCUSSION THREAD

**Discussion by geragray** [realm/realm-dotnet#2158] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rodion-m** [comment]
Is it ok that a button New (discussion) is not working?

**Reply by rodion-m** [reply]
So, then I ask my qestion here. I was wondering if I update Realm from 4.3 to 10, my db will be working fine? And what else can broke?

**Reply by nirinchev** [reply]
You can start a new discussion by clicking the "New discussion" button here: https://github.com/realm/realm-dotnet/discussions. If that's not working, it might be an issue with Github. I opened a new discussion on your behalf here: https://github.com/realm/realm-dotnet/discussions/2299.

**Reply by rodion-m** [reply]
The New button is like inactive.
Just https://github.com/realm/realm-dotnet/discussions/ page opens.

**Reply by rodion-m** [reply]
See: https://i.imgur.com/BnMNj6E.gif

**Reply by geragray** [reply]
@rodion-m it seems like by default discussion are limited only to users with full read/write privileges (https://github.com/github/feedback/discussions/29). Apologies that we didn't double-check how it works. I have enabled the setting that should enable all users with reading access to create discussions in Realm's repositories. 

Could you please verify it can start a new discussion now?

**Reply by rodion-m** [reply]
> @rodion-m it seems like by default discussion are limited only to users with full read/write privileges ([github/feedback#29](https://github.com/github/feedback/discussions/29)). Apologies that we didn't double-check how it works. I have enabled the setting that should enable all users with reading access to create discussions in Realm's repositories.
> 
> Could you please verify it can start a new discussion now?

Yes, the button is appeared now, it works. Thank you!
